source/fundamentals/serialization.txt

```diff
@@ -140,6 +140,54 @@ Implementing the `IBsonArraySerializer
 interface enables the driver to access serialization information for individual
 items in an array.
 
+.. _csharp-convention-packs:
+
+Convention Packs
+----------------
+
+Convention packs allow you to define and apply
+**conventions** to your classes and their members during serialization.
+Conventions modify how data is mapped between your
```

Comment:
S: if you want to keep modify here, consider including modify  from what? the default?

```suggestion
Conventions specify how data is mapped between your
```